yb-voyager/src/queryparser/queryParser.go

```diff
@@ -0,0 +1,33 @@
+package queryparser
+
+import (
+	pg_query "github.com/pganalyze/pg_query_go/v5"
+)
+
+type QueryParser struct {
```

Comment:
Yeah based on the current state of code we can avoid separate class.
But i think it will mature over time, and will become more complex. So i went ahead with separate class.

We can get rid of it in future if don't feel the need. What do you say?

Reply:
Okay to leave it in for now. But let's discuss along with @priyanshi-yb to figure out the right way to structure this package! 

On a related note, this is a nice read, something I'm trying to incorporate more :) - https://arpit.substack.com/i/147074725/premature-abstractions

Reply:
@sanyamsinghal are we keeping this as is for now?

Reply:
yes @priyanshi-yb